pkg/compute/init.go

```diff
@@ -185,19 +190,28 @@ func (c *InitCommand) Exec(in io.Reader, out io.Writer) (err error) {
 
 	progress.Step("Initializing package...")
 
-	if err := language.Initialize(progress); err != nil {
-		fmt.Println(err)
-		return fmt.Errorf("error initializing package: %w", err)
+	if language.Name != "other" {
+		if err := language.Initialize(progress); err != nil {
+			fmt.Println(err)
+			return fmt.Errorf("error initializing package: %w", err)
+		}
 	}
 
 	progress.Done()
 
 	text.Break(out)
 
-	text.Description(out, fmt.Sprintf("Initialized package %s to", text.Bold(m.Name)), abspath)
-	text.Description(out, "To publish the package (build and deploy), run", "fastly compute publish")
+	text.Description(out, fmt.Sprintf("Initialized project %s to", text.Bold(m.Name)), abspath)
+
+	if language.Name == "other" {
+		text.Description(out, "To package a pre-compiled WASM binary for deployment, run", "fastly compute pack")
+		text.Description(out, "To deploy the package, run", "fastly compute deploy")
+	} else {
+		text.Description(out, "To publish the package (build and deploy), run", "fastly compute publish")
+	}
+
 	text.Description(out, "To learn about deploying Compute@Edge projects using third-party orchestration tools, visit", "https://developer.fastly.com/learning/integrations/orchestration/")
-	text.Success(out, "Initialized package %s", text.Bold(m.Name))
+	text.Success(out, "Initialized project %s", text.Bold(m.Name))
```

Comment:
Why have we changed the terminology here from package to project? I personally feel this is out of the scope of this PR and we need to have a wider conversation with product and UX before changing our terminology as this will need to change in many other places in the user journey.